crates/fluvio/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "fluvio"
-version = "0.12.4"
+version = "0.13.0"
```

Comment:
is there need to bump to full version rather than patch

Reply:
I think that no, but I will get back to the previous version, but we dont need to patch more because 0.12.4 is not released yet